api/v1/routes/auth.py

```diff
@@ -110,3 +106,20 @@ def register_user(user: UserCreate, db: Session = Depends(get_db)):
 def read_admin_data(current_admin: Annotated[User, Depends(get_current_admin)]):
     return {"message": "Hello, admin!"}
 
+
+@auth.post("/password-reset-email", status_code=status.HTTP_200_OK)
+async def password_reset_email(request: PasswordResetRequest, db: Session = Depends(get_db)):
+    # Verify if the email exists in the database
+    user = db.query(User).filter(User.email == request.email).first()
+    if not user:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Email not found")
+
+    # Generate a unique token for password reset using JWT
+    token_data = {"sub": str(user.email)}
+    token = create_access_token(token_data)
+
+    # Send the password reset email with the reset link
+    reset_link = f"{settings.RESET_PASSWORD_URL}?token={token}"
+    send_password_reset_email(user.email, reset_link)
+
+    return {"message": "Password reset email sent successfully.", "reset_link": reset_link}
```

Comment:
There's a `JsonResponseDict` class to be used in standardizing the response format according to the specified convention, check it out, It's in the `utils.json_response` module